app/views/applications/show.html.erb

```diff
@@ -88,6 +88,25 @@
   <% end %>
 <% end %>
 
+<% if @github_available %>
+
```

Comment:
Nitpick: no purpose to this line of padding:

```suggestion
```

Reply:
Good spot!  Thanks.